openaps/vendors/dexcom.py

```diff
@@ -88,3 +95,31 @@ def main (self, args, app):
         break
     return records
 
+
+@use( )
+class iter_glucose_hours (glucose):
+  """ read last <hours> of glucose records, default 1, eg:
+
+* iter_glucose_hours     - read last 1 hour of glucose records
+* iter_glucose_hours 4.3 - read last 4.3 hours of glucose records
+  """
+  
+  def get_params (self, args):
+    return dict(hours=float(args.hours))
+  
+  def configure_app (self, app, parser):
+    parser.add_argument('hours', type=float, nargs='?', default=1,
+                        help="Number of hours of glucose records to read.")
+
+  def main (self, args, app):
+    params = self.get_params(args)
+    records = [ ]
+    for item in self.dexcom.iter_records('EGV_DATA'):
+      records.append(item.to_dict( ))
+      latest_time = dateutil.parser.parse(records[0]["system_time"])
+      earliest_time = dateutil.parser.parse(records[-1]["system_time"])
+      time_delta = (latest_time - earliest_time)
+      td = time_delta.seconds/3600.0 #convert to hours
```

Comment:
Since this is a `datetime.timedelta` object, you should use the `total_seconds()` method here.